### ISSUE TITLE
Primary email is not visually marked as primary

### ISSUE THREAD

**Issue by magnolia61** [civicrm/org.civicrm.contactlayout#119]
For phone numbers a visual sign is shown to show which is the primary phone number.
For email addresses this is not the case. Not sure this is a contactlayout thing though.

What would be needed to show which email address is the primary one?

without shoreditch
![image](https://user-images.githubusercontent.com/2195908/170932708-81876bb9-e1ca-41c3-8c19-9c65ea386503.png)

with shoredich
![image](https://user-images.githubusercontent.com/2195908/170933137-f6aaaa8d-29d7-43f0-8664-3fb8a2c0f47c.png)

**Comment by colemanw** [comment]
@magnolia61 I agree it's inconsistent. But isn't primary always the first one in the list?
